program-runtime/src/invoke_context.rs

```diff
@@ -277,6 +277,8 @@ pub trait InvokeContext {
     fn remove_first_keyed_account(&mut self) -> Result<(), InstructionError>;
     /// Get the list of keyed accounts
     fn get_keyed_accounts(&self) -> Result<&[KeyedAccount], InstructionError>;
+    /// Get the list of keyed accounts skipping `first_instruction_account` many entries
```

Comment:
Yes, it includes the `program_id` (at index 0).
And good point, I will update the comment.

Reply:
Thanks! It would be really helpful if the comment made it clearer what the breakdown of invoke context keyed accounts is and when it's appropriate to use `get_instruction_keyed_accounts` over `get_keyed_accounts`.